client/lib/service-worker/service-worker.js

```diff
@@ -1,24 +1,18 @@
-/**
- * /* eslint-disable no-console
- *
- */
-
 /**
  * WARNING: DO NOT USE ES2015+ OR COMMONJS. This file is served as-is and isn't
```

Comment:
Side note: we can probably update a lot of these warnings since we are running Node 16 and no longer support older browsers. So newer ES syntax should now work in most of these files